### PR TITLE
Edits to the visualization module

### PR DESCRIPTION
**Pull request by travisdriver**
- added `frustrum_ray_len` as an input variable so that it may be
adjusted for large-scale scenes
- removed references to the `MayAvi` library, as it's not supported
anymore

### CODE REVIEW

File: visualization/open3d_vis_utils.py [hunk 1]
```diff
@@ -96,7 +98,7 @@ def create_all_frustums_open3d(wTi_list: List[Pose3], calibrations: List[Cal3Bun
         py = K[1, 2]
         img_w = px * 2
         img_h = py * 2
-        frustum_obj = ViewFrustum(fx, img_w, img_h)
+        frustum_obj = ViewFrustum(fx, img_w, img_h, frustum_ray_len=frustrum_ray_len)
```
Comment (johnwlambert) [inline visualization/open3d_vis_utils.py:101]: thanks for the PR, @travisdriver. It looks good, except for a small typo on `frustum` here and throughout these two files

File: visualization/view_scene.py [hunk 2]
```diff
@@ -113,9 +95,13 @@ def view_scene(args: argparse.Namespace) -> None:
         default=0.1,
         help="if points are rendered as spheres, then spheres are rendered with this radius.",
     )
+    parser.add_argument(
+        "--frustrum_ray_len",
```
Comment (johnwlambert) [inline visualization/view_scene.py:99]: nit: `frustum`

File: visualization/view_scene.py [hunk 3]
```diff
@@ -113,9 +95,13 @@ def view_scene(args: argparse.Namespace) -> None:
...
+        type=float,
+        default=0.3,
+        help="Length extend frustum rays away from optical center "
```
Comment (johnwlambert) [inline visualization/view_scene.py:102]: nit: `Length to extend frustum rays ... `

File: visualization/open3d_vis_utils.py [hunk 4]
```diff
@@ -173,7 +175,7 @@ def draw_scene_open3d(
         calibrations: calibration object for each camera.
         args: rendering options.
     """
-    frustums = create_all_frustums_open3d(wTi_list, calibrations)
+    frustums = create_all_frustums_open3d(wTi_list, calibrations, args.frustrum_ray_len)
```
Comment (johnwlambert) [inline visualization/open3d_vis_utils.py:178]: nit: `frustum`

**Review by johnwlambert — CHANGES_REQUESTED** [review]
a few minor comments on variable names

File: visualization/open3d_vis_utils.py (hunk 4 above)
Comment (travisdriver) [reply visualization/open3d_vis_utils.py]: fixed

File: visualization/view_scene.py (hunk 3 above)
Comment (travisdriver) [reply visualization/view_scene.py]: fixed

File: visualization/view_scene.py (hunk 2 above)
Comment (travisdriver) [reply visualization/view_scene.py]: fixed

File: visualization/open3d_vis_utils.py (hunk 1 above)
Comment (travisdriver) [reply visualization/open3d_vis_utils.py]: fixed

**Review by johnwlambert — APPROVED** [review]

**Comment by johnwlambert** [comment]
Nice fix Travis, thanks

**Pull request merged**